process/block/postprocess/basePostProcess.go

```diff
@@ -20,6 +20,7 @@ var _ process.DataMarshalizer = (*basePostProcessor)(nil)
 type txShardInfo struct {
 	senderShardID   uint32
 	receiverShardID uint32
+	index           uint32
```

Comment:
I think index could be moved in upper DTO -> txInfo, as it is related to tx not to shard info